lib/spell/role/session.ex

```diff
@@ -203,4 +204,11 @@ defmodule Spell.Role.Session do
     end
   end
 
+  @spec config_timeout() :: integer
+  defp config_timeout do
+    case Application.get_env(:spell, :timeout) do
```

Comment:
Allowing timeouts to be configured is great! One suggestion: [`Application.get_env/3`](http://elixir-lang.org/docs/stable/elixir/Application.html#get_env/3) allows for a default argument, which would slightly simplify the code.